packages/next/package.json

```diff
@@ -60,7 +60,7 @@
     ]
   },
   "dependencies": {
-    "@ampproject/toolbox-optimizer": "2.6.0",
+    "@ampproject/toolbox-optimizer": "2.7.0-alpha.1",
```

Comment:
This gets flagged for a vulnerability further down. Needs to be `2.7.0-alpha.4` to avoid the vulnerable `node-fetch` version.